### ISSUE TITLE
New python code (filters)

### ISSUE THREAD

**Issue by FerCris** [3DLIRIOUS/MeshLabXML#12]
Hello,

I've seen your work and I think it is very useful for our research projects as students. However, I've tried to do something else regarding other filters I have not found, such as 'distance from reference mesh' and the tool 'align to principal axis', for example. So, my question is if there is any way to find this code; Has it been already implemented?

Thanks in advance,
FerCris

**Comment by 3DLIRIOUS** [comment]
No, these filters haven't been added yet. I can try to add them in the near future, or you can try to add them yourself (it's not too difficult).

**Comment by FerCris** [comment]
Ok, thanks for your quick answer. I'm working now on this, if I reach any results I will upload it.

**Comment by fishdda** [comment]
Is there any progress for calculating the curvatures?
